### DISCUSSION TITLE
How can I show more than 10 results in admin collection screen?

### DISCUSSION THREAD

**Discussion by raguila8** [tinacms/tinacms#3030] [Help] [answered]
Hi, I have a collection page with more than 10 items. But it looks like the admin page for collections shows a max of 10 items. I want to be able to show all of the items. How can I do this?

![CleanShot 2022-07-04 at 11 57 14](https://user-images.githubusercontent.com/29444224/177206751-5bf477fc-5318-4b6c-9869-6e20ff5cf4cb.png)

**Comment by jamespohalloran** [comment, marked as answer]
That seems like a bug! We have been working on our new "data layer" feature, which is currently an opt in thing. If you enable it both in the CLI, and in cmsCallback, you should have the ability to paginate in the UI: https://tina.io/docs/tina-cloud/data-layer/#enabling-the-data-layer

**Reply by raguila8** [reply]
This solved it. I had the `--experimentalData`  flag set in the cli, but not in the cmsCallback. Once, I added it to the cmsCallback, the results are paginated.

Thanks!
